sdk/cosmos/mgmt/src/main/java/com/azure/management/cosmosdb/implementation/CosmosDBAccountImpl.java

```diff
@@ -622,7 +603,7 @@ private void addLocationsForParameters(HasLocations locationParameters, List<Fai
                                         .flatMap(
                                             index -> {
                                                 data.set(0, data.get(0) + 30);
-                                                return Mono.delay(manager().getSdkContext().getLroRetryDuration());
+                                                return Mono.delay(SdkContext.getLroRetryDuration());
```

Comment:
Why it using lro regry duration?

Reply:
The cosmos multi location prevision is async. It will return succeeded when first location is done. This func I think is to ensure all location is succeeded.

Reply:
Yes, but why use `getLroRetryDuration`? It should be only used for LRO, but this is not.

Reply:
You can use `getDelayDuration` with a specified delay.

Reply:
This is a LRO situation, just isn't a common LRO situation.

Reply:
Long running operation to wait all location prevision complete.

Reply:
An operation takes long time does not means that it is LRO.

The specific var is used here for the `PollerFlux`.

```
   public <T, U> PollerFlux<PollResult<T>, U> getLroResultAsync(Mono<SimpleResponse<Flux<ByteBuffer>>> lroInit,
                                                                 HttpPipeline httpPipeline,
                                                                 Type pollResultType, Type finalResultType) {
        return PollerFactory.create(
            getSerializerAdapter(),
            httpPipeline,
            pollResultType,
            finalResultType,
            SdkContext.getLroRetryDuration(),
            activationOperation(lroInit)
        );
    }
```

Reply:
Track1 is such

```
                        ).repeatWhen(new Func1<Observable<? extends java.lang.Void>, Observable<?>>() {
                            @Override
                            public Observable<?> call(Observable<? extends Void> observable) {
                                data.set(0, data.get(0) + 5);
                                return observable.delay(5, TimeUnit.SECONDS);
                            }
                        })
```

Reply:
Done, change to `getDelayDuration`.